### DISCUSSION TITLE
Run code when filename is given

### DISCUSSION THREAD

**Discussion by vfacta** [quarto-dev/quarto-cli#5509] [Q&A] [answered]
Currently, when a filename is given in a code environment, it seems not to be possible to display the output from the as when no filename is given. For example, the code below displays its output wheter in html or pdf :

````
```{python}
print("hello, world!")
```
````

However, the next following code won't display its output:

````
```{.python filename="helllo.py" }
print("hello, world!")
```
````
It would be great if there was an option for choosing to run the code when it has a filename. Also, parameters such as `#| output: true` inside the code above are not recognized as parameters, being regarded as code commentaries and printed as such.

**Comment by mcanouil** [comment, marked as answer]
The dot prefix before the language means no computation: ` ```{.language}` is equivalent to ` ```language`.
This is a feature, thus if you use the dot prefix and brackets, you'll never see outputs and code cells parameters won't have effects, because it's not a code cell, it's a regular code block.

**Reply by vfacta** [reply]
Very helpful! Wasn't aware about the dot
